app/components/PredictionMarkets/PMAssetsContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import AssetStore from "stores/AssetStore";
+import MarketsStore from "stores/MarketsStore";
+import AccountStore from "stores/AccountStore";
+import AltContainer from "alt-container";
+import PredictionMarkets from "./PredictionMarkets";
+import BindToChainState from "../Utility/BindToChainState";
+
+class PMAssetsContainer extends React.Component {
+    render() {
+        return (
+            <AltContainer
+                stores={[AssetStore, AccountStore, MarketsStore]}
+                inject={{
```

Comment:
please restrict to the ones actually in use

Reply:
Done